projects/baby_driver/scripts/first.py

```diff
@@ -0,0 +1,4 @@
+"""thin wrapper over can_message"""
+from scripts import can_message
+
+can_message.send_message(1)
```

Comment:
This file is just for testing right? not sure if necessary

Reply:
Oh yeah. It's just a sample in case we wanted to change from just using can_message.py and instead have wrappers that just call the specific function callback we want. I can remove if you want.